webapp/src/components/SafeConfigDisplay.test.tsx

```diff
@@ -0,0 +1,41 @@
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { describe, expect, it } from "vitest";
+import SafeConfigDisplay from "./SafeConfigDisplay";
+
+describe("SafeConfigDisplay", () => {
+	const config = {
+		owners: ["0xOwner1", "0xOwner2"],
```

Comment:
I wonder if it makes sense to use legal values here (in case some processing is done on the values - like adding ellipsis or doing ENS stuff).